File: SoloLearn_CodeCoach/Easy/Hovercraft/Hovercraft.java
import java.util.Scanner;

public class Hovercraft {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int sales = input.nextInt();

        int perMonth = (10 * -2_000_000 - 1_000_000) + sales * 3_000_000;

        if (perMonth < 0) {
            System.out.println("Loss");
        } else if (perMonth > 0) {
            System.out.println("Profit");
        } else {
            System.out.println("Broke Even");
        }

    }
}

File: SoloLearn_CodeCoach/Easy/Hovercraft/Hovercraft.cpp
#include <iostream>
using namespace std;

int main() {
    int sales;
    cin >> sales;

    int per_month = -1 * (10 * 2000000 + 1000000) + sales * 3000000;

    if (per_month < 0) {
        cout << "Loss" << endl;
    } else if (per_month > 0) {
        cout << "Profit" << endl;
    } else {
        cout << "Broke Even" << endl;
    }

    return 0;
}
